HTCONDOR-2724 Update authentication map file docs (#2952)
htcondor/htcondor: docs/admin-manual/security.rst

list of authentication methods for testing purposes only.
:index:`authentication`

The Authentication Map File
---------------------------

:index:`authentication map file`

> HTCondor's authentication map file allows you to define how a name
> produced by a given authentication method are translated into an

Reviewer: Singular/plural agreement; the HT is silent.
```suggestion
HTCondor's authentication map file allows you to define how a name
produced by a given authentication method is translated into a
```
